server/src/main/java/com/cloud/projects/ProjectManagerImpl.java

```diff
@@ -479,7 +479,7 @@ public void doInTransactionWithoutResult(TransactionStatus status) throws Resour
                 throw new InvalidParameterValueException("Unable to find account name=" + newOwnerName + " in domain id=" + project.getDomainId());
             }
             Account currentOwnerAccount = getProjectOwner(projectId);
-            if (currentOwnerAccount.getId() != futureOwnerAccount.getId()) {
+            if (currentOwnerAccount !=null && currentOwnerAccount.getId() != futureOwnerAccount.getId()) {
                 ProjectAccountVO futureOwner = _projectAccountDao.findByProjectIdAccountId(projectId, futureOwnerAccount.getAccountId());
```

Comment:
Here, I think when currentOwnerAccount ==null, we should throw InvalidParameterValueException.